src/plugins_3rdparty/primer3/src/Primer3Dialog.ui

```diff
@@ -6,8 +6,8 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>1199</width>
-    <height>814</height>
+    <width>1024</width>
+    <height>768</height>
```

Comment:
A lot if people still uses 1024x768 monitors, it is not good to make this dialog larger